[2025-05-09T16:16:45.114+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_IBGE.salvar_dados_em_tabela manual__2025-05-09T16:16:29.293540+00:00 [queued]>
[2025-05-09T16:16:45.122+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_IBGE.salvar_dados_em_tabela manual__2025-05-09T16:16:29.293540+00:00 [queued]>
[2025-05-09T16:16:45.122+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-05-09T16:16:45.164+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): salvar_dados_em_tabela> on 2025-05-09 16:16:29.293540+00:00
[2025-05-09T16:16:45.169+0000] {standard_task_runner.py:60} INFO - Started process 517 to run task
[2025-05-09T16:16:45.171+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'elt_IBGE', 'salvar_dados_em_tabela', 'manual__2025-05-09T16:16:29.293540+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp8a5rp1sk']
[2025-05-09T16:16:45.174+0000] {standard_task_runner.py:88} INFO - Job 13: Subtask salvar_dados_em_tabela
[2025-05-09T16:16:45.247+0000] {task_command.py:423} INFO - Running <TaskInstance: elt_IBGE.salvar_dados_em_tabela manual__2025-05-09T16:16:29.293540+00:00 [running]> on host 59e4007f2f7a
[2025-05-09T16:16:45.939+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_IBGE' AIRFLOW_CTX_TASK_ID='salvar_dados_em_tabela' AIRFLOW_CTX_EXECUTION_DATE='2025-05-09T16:16:29.293540+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-09T16:16:29.293540+00:00'
[2025-05-09T16:16:45.951+0000] {base.py:83} INFO - Using connection ID 'meu_postgres' for task execution.
[2025-05-09T16:16:46.255+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-09T16:16:46.268+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=elt_IBGE, task_id=salvar_dados_em_tabela, execution_date=20250509T161629, start_date=20250509T161645, end_date=20250509T161646
[2025-05-09T16:16:46.313+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-09T16:16:46.331+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
